django/contrib/admin/utils.py

```diff
@@ -490,10 +490,20 @@ def construct_change_message(form, formsets, add):
     Translation happens later on LogEntry access.
     """
     change_message = []
+    changed_data = form.changed_data
+    changed_field_labels = []
+    with translation_override(None):
```

Comment:
IMO, this block merits a comment .... "Deactivate translations  and (do something)."